### ISSUE TITLE
AVT scan violation: Form control has more than one label

### ISSUE THREAD

**Issue by ttliang** [carbon-design-system/carbon#9389]
IBM Component are generating label elements automatically e.g. ibm-search, ibm-pagination, ibm-radio. Labels triggered AVT scan violation "Form control has more than one label." when we apply these components into one page.
## Environment
> MacOS v11.4
> Firefox  v90.0.2 (64-bit)
> IBM Accessibility Equal Access Accessibility Checker Version 3.1.9
> carbon-components-angular v4.53.1

**Comment by tw15egan** [comment]
Hey there @ttliang, can you recreate the error in a [Codesandbox](https://codesandbox.io/s/inspiring-fermat-vg0tk) so we can take a look? I'm unable to reproduce the error in our [Storybook](https://react.carbondesignsystem.com/iframe.html?id=components-datatable--with-toolbar&args=&viewMode=story). The `for` attribute is correct, and is not a duplicate ID, as it shows the relationship between the input and labels element for screenreader accessibility.

**Comment by ttliang** [comment]
Confirmed it's not a bug of carbon, thus close it.
